lib/ansible/plugins/inventory/vultr.py

```diff
@@ -0,0 +1,99 @@
+# (c) 2018, Yanis Guenane <yanis+ansible@guenane.org>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+    name: vultr
+    plugin_type: inventory
+    author:
+      - "Yanis Guenane (@Spredzy)"
+    short_description: Vultr inventory source
+    description:
+        - Get inventory hosts from Vultr public cloud.
+        - Uses ~/.vultr.ini, ./vultr.ini or VULTR_API_CONFIG path to config file.
+    options:
+        plugin:
+            description: token that ensures this is a source file for the 'vultr' plugin.
+            required: True
+            choices: ['vultr']
+        api_account:
+            description: Specify the account to be used.
+'''
+
+EXAMPLES = r'''
+# vultr_inventory.yml file in YAML format
+# Example command line: ansible-inventory --list -i vultr_inventory.yml
+
+plugin: vultr
+'''
+
+import json
+import os
+
+from ansible.errors import AnsibleError
+from ansible.plugins.inventory import BaseInventoryPlugin
+from ansible.module_utils.six.moves import configparser
+from ansible.module_utils.urls import open_url
+from ansible.module_utils._text import to_native
+
+
+def _retrieve_api_key(api_account):
```

Comment:
You should use an `api_key` plugin option ([example](https://github.com/ansible/ansible/blob/81ca04512dbe52f6057895a2525484bc5b60a837/lib/ansible/plugins/inventory/scaleway.py#L33)) and an `api_config` option (type: `path`). As spotted by resmo, it seems that [Vultr.read_ini_config](https://github.com/ansible/ansible/blob/46323dcdac55b5d89c5803dc38c01f80ed2eb0e1/lib/ansible/module_utils/vultr.py#L96) should be reused.

Reply:
Done.